reference-architecture/vmware-ansible/roles/rhn-subscription/tasks/main.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Is the host already registered?
+  command: "subscription-manager list"
+  register: subscribed
+  ignore_errors: yes
+
+- name: Register host via Activation key
+  redhat_subscription:
+    activationkey: "{{ rhsm_activation_key }}"
+    org_id: "{{ rhsm_org_id }}"
+    state: present
+    pool: "^(60 Day Supported OpenShift Enterprise|OpenShift Enterprise, Standard|OpenShift Enterprise, Premium|Employee)"
```

Comment:
@cooktheryan @dav1x might be a good idea to take shared roles like this and move them up to the root of openshift-ansible-contrib instead of duplicating in each ref-arch.